surveys/forms.py

```diff
@@ -146,7 +146,10 @@ def __init__(self, *args, **kwargs):
         super(StudyCreateForm, self).__init__(*args, **kwargs)
         self.create_default_helper()
         self.fields['areas'].widget.attrs['class'] = 'basic-multiple'
+
         self.fields['agency'].initial = pldp_models.Agency.objects.first()
+        self.fields['agency'].queryset = pldp_models.Agency.objects.filter(is_active='t')
```

Comment:
Forgot to raise this in the course of reviewing #194: What's the advantage of filtering on `is_active='t'` instead of `is_active=True`?

Reply:
It's just the style I've been using! That's how it shows up in psql so it made sense to me, and then I kept it consistent. Does it make a difference?

Reply:
Gotcha! I couldn't find any docs on what specific values would evaluate to `True` when filtering on a BooleanField in the Django ORM but it makes sense that `'t'` would work. From a stylistic perspective using the `True` boolean makes a bit more sense to me than `'t'` since the ORM is supposed to be an abstraction layer that serializes Postgres records into Python objects, and `True` is the Pythonic boolean. But since `'t'` works in this case I think it's a question of taste.

If you're curious, I found a thread on the Django dev google group discussing this behavior: https://groups.google.com/forum/#!topic/django-developers/f82vcCMrpSs Looks like anything truthy will evaluate to `True`, and anything falsey will evaluate to `False`. Some interesting discussions of edge cases in there.

Reply:
Makes sense! I'll use `True` next time.